File: src/fiefapp.svelte/src/models/village.ts
interface IVillage {
  id: string;
  name: string;
  serfdoms: number;
  farmers: number;
  boatbuilders: number;
  tanneres: number;
  millers: number;
  furriers: number;
  tailors: number;
  blacksmiths: number;
  carpenters: number;
  innkeepers: number;
}

export class Village implements IVillage {
  id: string;
  name: string;
  serfdoms: number;
  farmers: number;
  boatbuilders: number;
  tanneres: number;
  millers: number;
  furriers: number;
  tailors: number;
  blacksmiths: number;
  carpenters: number;
  innkeepers: number;

  constructor() {
    this.id = '';
    this.name = 'Ny by';
    this.serfdoms = 0;
    this.farmers = 0;
    this.boatbuilders = 0;
    this.tanneres = 0;
    this.millers = 0;
    this.furriers = 0;
    this.tailors = 0;
    this.blacksmiths = 0;
    this.carpenters = 0;
    this.innkeepers = 0;
  }
}

File: src/fiefapp.entities/Village.cs
using fiefapp.entities.Interfaces;
using System;

namespace fiefapp.entities
{
    public class Village : IBaseEntity, IFiefForeinKey
    {
        public Guid Id { get; set; }
        public Guid FiefId { get; set; }
        public string Name { get; set; }
        public int Serfdoms { get; set; }
        public int Farmers { get; set; }
        public int Boatbuilders { get; set; }
        public int Tanners { get; set; }
        public int Millers { get; set; }
        public int Furriers { get; set; }
        public int Tailors { get; set; }
        public int Blacksmiths { get; set; }
        public int Carpenters { get; set; }
        public int Innkeepers { get; set; }
    }
}
